Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM docker/compose:alpine-1.29.2
```

Comment:
Needed?

Now that you did the magic with the `action.yml`, I think that half the files in the repo can be deleted.  e.g. both Dockerfiles, docker-compose.yml, etc. etc.

Reply:
I think you're not on the most up to date branch, I've already removed this